CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/StaffLoginPage.java

```diff
@@ -70,4 +76,10 @@ public void successfulLogin(){     //replace with orders list class when availab
             Intent intent = new Intent(this, MainActivityLoginPage.class);
             startActivity(intent);
         }
-    }
\ No newline at end of file
+
+
+
+    }
+
+    }
```

Comment:
The tests point at this line number 84, where the last bracket is, "error: class, interface, or enum expected".

Reply:
Thank you! i will change it